docs/general/builtins/scalar-functions.rst

```diff
@@ -2446,6 +2446,44 @@ null_string
 If the ``null_string`` argument is omitted or NULL, none of the substrings of
 the input will be replaced by NULL.
 
+.. _scalar-array-min:
+
+``array_min(array)``
+--------------------
+
+The ``array_min`` function returns the smallest element in ``array``. If
+``array`` is ``NULL`` or an empty array, the function returns``NULL``.
+Supports arrays of any of :ref:`sql_ddl_datatypes_primitives`.
```

Comment:
```suggestion
The ``array_min`` function returns the smallest element in ``array``. If
``array`` is ``NULL`` or an empty array, the function returns ``NULL``. This
function supports arrays of any of the :ref:`primitive types
<sql_ddl_datatypes_primitives>`.
```

- corrected missing space before `NULL`
- added "This function" so the sentence is in the active voice (i.e., the do-er is explicit)
- added explicit link text for `sql_ddl_datatypes_primitives` (it's best to avoid relying on the heading text of the link target being or remaining suitable)
- removed the double space (`  `) after the period (`.`) because spaces are collapsed by RST so double spaces only introduce diff noise
- rewrapped to 79 chars

Reply:
Thanks for corrections!